lib/potluck.rb

```diff
@@ -0,0 +1,37 @@
+require './lib/dish.rb'
+
+class Potluck
+  attr_accessor :date, :dishes, :category
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(category_type)
+    @category[category_type]
```

Comment:
You can get the category from a dish object by calling the .category method on a Dish instance. So for your array that is holding dish instances, @dishes, you could get a single dishes category with @dishes.first.category